indico/htdocs/sass/partials/_corner-messages.scss

```diff
@@ -62,17 +62,13 @@
     @extend %font-family-title;
     @include border-radius(2px);
     @include border-all($light-black);
-    @include transition(background-color .5s, color .5s, border-color .5s, width .5s);
-    @include indico-animation(_fadein, .5s, normal forwards ease-in-out, 1);
+    @include transition(background-color 0.5s, color 0.5s, border-color 0.5s, width 0.5s);
+    @include indico-animation(_fadein, 0.5s, normal forwards ease-in-out, 1);
     @include _progress(darken($light-gray, 0.3*$color-variation), $light-gray);
 
-    display: block;
-    float: right;
-    clear: both;
-
     padding: 1.2em;
     margin-bottom: 1em;
-    box-shadow: 0px 3px 2px rgba(150, 150, 150, 0.2);
+    box-shadow: 0 3px 2px rgba(150, 150, 150, 0.2);
```

Comment:
We have the `box-shadow` mixin, check how it's used (ex: `/indico/htdocs/sass/partials/_forms.scss`)